src/Resolver/LayoutResolver/LayoutPathResolver.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Efabrica\PHPStanLatte\Resolver\LayoutResolver;
+
+final class LayoutPathResolver
+{
+    public function resolve(?string $templatePath): ?string
+    {
+        if ($templatePath === null) {
+            return null;
+        }
+
+        $templateContent = file_get_contents($templatePath) ?: '';
+        preg_match('/{(layout|extend) (?<layout_name>.*?)}/', $templateContent, $match);
```

Comment:
Could there be a better way to solve this than analsis of template text by regexp? Something similar to mechanism we use in included templates? Analyse compiled template source a find nodes that rewrites layout and use default layout (passed in something like $__layout__)?